src/main/java/io/antmedia/rest/StreamsSourceRestService.java

```diff
@@ -60,7 +65,7 @@
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Path("/addStreamSource")
 	@Produces(MediaType.APPLICATION_JSON)
-	public Result addStreamSource(Broadcast stream) {
+	public Result addStreamSource(Broadcast stream, @QueryParam("socialNetworks") String socialEndpointIds) {
```

Comment:
I think we need functional test code for this feature. What do you think? @burak-58

Reply:
Yes, I think so.